### ISSUE TITLE
[IDEA] Implement `focusFromStart` and `focusFromEnd` for `tm-focus`

### ISSUE THREAD

**Issue by yaisog** [TiddlyWiki/TiddlyWiki5#7338]
Like I wrote in [this comment](https://github.com/Jermolene/TiddlyWiki5/pull/7222#issuecomment-1410194593), it would be nice if `tm-focus` could be amended to also accept the `focusFromStart` and `focusFromEnd` parameters implemented in #7222 (when targeting text inputs).

**Comment by Jermolene** [comment]
Hi @yaisog thank you, that makes sense.

**Comment by yaisog** [comment]
Hi @Jermolene , would you like to pursue a PR? 😂

**Comment by Jermolene** [comment]
Thank you @yaisog please do!

**Comment by yaisog** [comment]
@Jermolene: No, wait! I didn't write "Would you like _me_ to pursue a PR"...

**Comment by pmario** [comment]
Reopen, since the idea was accepted as a worthwhile improvement. It got an actionable flag, which means someone can implement it.

@yaisog ... It's not the author of an idea, that has to implement it. It can be anyone.
